apps/src/Observer.js

```diff
@@ -0,0 +1,52 @@
+/** @file Helper for consuming ObservableEvents. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
```

Comment:
I'll comment these.  This is the same, stricter rules applied throughout NetSim code - I find it useful to enforce them on new files.  `unused` and `eqeqeq` in particular catch stuff all the time for me.